docs/introduction/basics-about-model-architecture.md

```diff
@@ -67,174 +67,7 @@ class Product
-
-    /**
-     * @var string|null
-     *
-     * @ORM\Column(type="text", nullable=true)
-     */
-    protected $seoTitle;
-
-    // ...
-
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Product\Brand\Brand $brand
-     * @param int $domainId
-     */
-    public function __construct(Brand $brand, $domainId)
-    {
-        $this->brand = $brand;
-        $this->domainId = $domainId;
-    }
-
-    /**
-     * @return string|null
-     */
-    public function getSeoTitle()
-    {
-        return $this->seoTitle;
-    }
-
-    /**
-     * @param string|null $seoTitle
-     */
-    public function setSeoTitle($seoTitle)
-    {
-        $this->seoTitle = $seoTitle;
-    }
-
-    // ...
-
-}
-```
-
-...and its main entity `Brand` working as a proxy:
-
-```php
-// FrameworkBundle/Model/Product/Brand/Brand.php
-
-namespace Shopsys\FrameworkBundle\Model\Product\Brand;
-
-/**
- * @ORM\Table(name="brands")
- * @ORM\Entity
- */
-class Brand extends AbstractTranslatableEntity
-{
-
-    // ...
-
-    /**
-     * @param int $domainId
-     * @return string
-     */
-    public function getSeoTitle(int $domainId)
-    {
-        return $this->getBrandDomain($domainId)->getSeoTitle();
-    }
-    
-    /**
-     * @param int $domainId
-     * @return \Shopsys\FrameworkBundle\Model\Product\Brand\BrandDomain
-     */
-    protected function getBrandDomain(int $domainId)
-    {
-        foreach ($this->domains as $domain) {
-            if ($domain->getDomainId() === $domainId) {
-                return $domain;
-            }
-        }
-
-        throw new BrandDomainNotFoundException($this->id, $domainId);
-    }
-
-    // ...
-
-}
-```
-
-## Entity Factory
-
-Is a class that creates an entity.
-The framework must allow using extended entities and this problem is solved using factories.
-
-The only entities that are not created by a factory are `*Translation` and `*Domain` entities.
-These entities are created by their main entity.
-
-### Example
-```php
-// FrameworkBundle/Model/Cart/Item/CartItemFactoryInterface.php
-
-namespace Shopsys\FrameworkBundle\Model\Cart\Item;
-
-// ...
-
-interface CartItemFactoryInterface
-{
-
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Customer\CustomerIdentifier $customerIdentifier
-     * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
-     * @param int $quantity
-     * @param string $watchedPrice
-     * @return \Shopsys\FrameworkBundle\Model\Cart\Item\CartItem
-     */
-    public function create(
-        CustomerIdentifier $customerIdentifier,
-        Product $product,
-        int $quantity,
-        string $watchedPrice
-    ): CartItem;
-}
-```
-
-The factory has an implementation in the framework and can be overwritten in your project when you need to work with an extended entity.
-You can read about entity extension in a [separate article](../wip_glassbox/entity-extension.md).
+You'll find more about our entities specialities in a [detailed article](entities.md).
```

Comment:
Please do not place important information below big image and next heading. It is hard to find thi link to this separate article. I would suggest to place it above example